mmv1/third_party/terraform/services/modelarmor/resource_model_armor_template_test.go

```diff
@@ -0,0 +1,250 @@
+
+		"filter_config_pi_and_jailbreak_filter_settings_filter_enforcement": "ENABLED",
+		"filter_config_pi_and_jailbreak_filter_settings_confidence_level":   "HIGH",
+
+		"filter_config_malicious_uri_filter_settings_filter_enforcement": "ENABLED",
+
+		"template_metadata_custom_llm_response_safety_error_message":                 "This is a custom error message for LLM response",
+		"template_metadata_log_template_operations":                                  true,
+		"template_metadata_log_sanitize_operations":                                  true,
+		"template_metadata_multi_language_detection_enable_multi_language_detection": true,
+		"template_metadata_ignore_partial_invocation_failures":                       true,
+		"template_metadata_custom_prompt_safety_error_code":                          400,
+		"template_metadata_custom_prompt_safety_error_message":                       "This is a custom error message for prompt",
+		"template_metadata_custom_llm_response_safety_error_code":                    401,
+		"template_metadata_enforcement_type":                                         "INSPECT_ONLY",
+	}
+
+	updatedContext := map[string]interface{}{
+		"location":   initialContext["location"],
+		"templateId": initialContext["templateId"],
+
+		"label_test_label": "env-updated-" + acctest.RandString(t, 5),
+
+		"filter_config_rai_settings_rai_filters_0_filter_type":      "DANGEROUS",
+		"filter_config_rai_settings_rai_filters_0_confidence_level": "LOW_AND_ABOVE",
+
+		"sdp_settings_config_type":                                       "basic_config",
+		"filter_config_sdp_settings_basic_config_filter_enforcement":     "ENABLED",
+		"filter_config_sdp_settings_advanced_config_inspect_template":    "",
+		"filter_config_sdp_settings_advanced_config_deidentify_template": "",
+
+		"filter_config_pi_and_jailbreak_filter_settings_filter_enforcement": "DISABLED",
+		"filter_config_pi_and_jailbreak_filter_settings_confidence_level":   "MEDIUM_AND_ABOVE",
+
+		"filter_config_malicious_uri_filter_settings_filter_enforcement": "DISABLED",
+
+		"template_metadata_custom_llm_response_safety_error_message":                 "Updated LLM error message",
+		"template_metadata_log_template_operations":                                  false,
+		"template_metadata_log_sanitize_operations":                                  false,
+		"template_metadata_multi_language_detection_enable_multi_language_detection": false,
+		"template_metadata_ignore_partial_invocation_failures":                       false,
+		"template_metadata_custom_prompt_safety_error_code":                          404,
+		"template_metadata_custom_prompt_safety_error_message":                       "Updated prompt error message",
+		"template_metadata_custom_llm_response_safety_error_code":                    500,
+		"template_metadata_enforcement_type":                                         "INSPECT_AND_BLOCK",
+	}
+
+	const config_template = `
```

Comment:
Generally we recommend putting the config in a separate function, similar to https://github.com/hashicorp/terraform-provider-google/blob/0ca0b4b187c09a36fe427f8cb5d3026c83a94910/google/services/osconfig/resource_os_config_patch_deployment_generated_test.go#L46, but this doesn't seem to be causing problems with our test parsing logic so it may be fine.